lib/sdk/iden3comm.dart

```diff
@@ -148,10 +148,14 @@ abstract class PolygonIdSdkIden3comm {
   ///
   /// The [interaction] is the interaction to be saved
   /// The [privateKey]  is the key used to access all the sensitive info from the identity
+  /// The [genesisDid] is the unique id of the identity
+  /// The [profileNonce] is the nonce of the profile used from identity
+  /// to obtain the did identifier
   Future<InteractionBaseEntity> addInteraction({
     required InteractionBaseEntity interaction,
     String? genesisDid,
     String? privateKey,
+    BigInt? profileNonce,
```

Comment:
profileNonce is not needed, as we are storing all interactions in genesisDid db

Reply:
Oki, removing it.
We also need to remove it from the Flutter side (I added it because abstract and impl where not sync, I chose the wrong one XD).

Reply:
Hum actually we are using the `profileNonce` in `AddInteractionUseCase`?
Should we also remove it there and use genesisDid profile?
Same with `removeInteraction`?